### PR TITLE
Changelog update for `v1.110.0` 

### PR DESCRIPTION
**Pull request by confused-Techie**
Here's our monthly changelog updates.
With the lack of changes that modify core functionality I've brought up if we want to not jump so far ahead in versioning this time around, just to see if others are on board.

But of course that being subject to change, this gets all of our changelog updates in for now.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,8 +5,57 @@
...
 ## [Unreleased]
+
+## 1.110.0
+
+- Made the modification of `editor.preferredLineLength` configurable within `wrap-guide` when changing `wrap-guide.columns`
+- Fixed Snippets from `language-php` that would lose the `$` character
+- Fixed a condition where an invalid config may crash Pulsar before fully starting up, but not registering that it's crashed
+- Reduced error notifications that may appear from `autocomplete-html` when handling EJS files
+- Fixed macOS binary signing after moving over to GitHub Actions for CI
+- Updated PPM to a newer `node-gyp` allowing newer versions of C++ toolkits to be used
```
Comment (DeeDeeG) [inline CHANGELOG.md:16]:
```suggestion
- Updated PPM to a newer `node-gyp`, allowing newer versions of C/C++ compiler toolchains and Python to be used (also dropped support for Python 2.x!)
```

File: packages/welcome/lib/changelog-view.js [hunk 2]
```diff
@@ -50,31 +50,25 @@ export default class ChangeLogView {
...
-                Fixed the icon used when registering Pulsar as a file handler on Windows.
+                Reduced error notifications that may appear from `autocomplete-html` when handling EJS files
               </li>
               <li>
-                Removed the non-functional `autoUpdate` API from Pulsar, instead relying on the `pulsar-updater` package.
+                Fixed macOS binary signing after moving over to GitHub Actions for CI
               </li>
               <li>
-                Ensure that project-specific setting overrides don't leak to the user's config file when the settings UI is visited.
+                Updated PPM to a newer `node-gyp` allowing newer versions of C++ toolkits to be used
```
Comment (DeeDeeG) [inline packages/welcome/lib/changelog-view.js:68]:
```suggestion
                Updated PPM to a newer `node-gyp`, allowing newer versions of C/C++ compiler toolchains and Python to be used (also dropped support for Python 2.x!)
```

**Review by DeeDeeG — COMMENTED** [review]
Maybe tweak the node-gyp bump changelog text?

**Comment by confused-Techie** [comment]
@DeeDeeG Got those changes in, thanks for catching that!

**Comment by DeeDeeG** [comment]
> @DeeDeeG Got those changes in, thanks for catching that!

I made the changes for `packages/welcome/lib/changelog-view.js` as well, looks like the change for that file is not committed?

File: packages/welcome/lib/changelog-view.js (hunk 2 above)
Comment (confused-Techie) [reply packages/welcome/lib/changelog-view.js:68]: Didn't even realize this was for a different file, assumed that GitHub was being weird lol.
Thanks for pointing it out

**Review by DeeDeeG — APPROVED** [review]
"Rubber-stamp approve" seems basically reasonable, content is text only so unlikely to have problems. (CI has been looking good.)

I haven't verified everything in here, but it's good to have a changelog! Approving formally for the process mostly.

**Pull request merged**